### ISSUE TITLE
[BITV] 9.1.3.1b/2.2 - The "statuses" list was implemented semantically incorrectly - the "li" elements are not implemented as children of the "ul" element, but are inside "div" elements or missing. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36917]
- [x] Fix
- [x] Backport

<img width="651" alt="ce11e7fc4d8f3e85d36561e86593ca97" src="https://user-images.githubusercontent.com/1315170/221931335-127ceda5-0b1d-4034-b83f-72f79919a04f.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1d5eadb58d-v2-n2

</details>

**Comment by JuliaKirschenheuter** [comment]
Checked it and seems to be right now:

![Screenshot from 2023-03-28 12-50-28](https://user-images.githubusercontent.com/6078378/228213372-0b315a9b-cd05-41f1-9038-c9cad1a91d94.png)

**Comment by JuliaKirschenheuter** [comment]
**have to check in stable25**
